src/main/scala/is/hail/HailContext.scala

```diff
@@ -239,22 +241,24 @@ class HailContext private(val sc: SparkContext,
     if (inputs.isEmpty)
       fatal(s"arguments refer to no files: '${ files.mkString(",") }'")
 
-    BgenLoader.load(this, inputs, sampleFile, tolerance, nPartitions)
+    BgenLoader.load(this, inputs, sampleFile, tolerance, nPartitions, mangleDuplicates)
   }
 
   def importGen(file: String,
     sampleFile: String,
     chromosome: Option[String] = None,
     nPartitions: Option[Int] = None,
-    tolerance: Double = 0.2): GenericDataset = {
+    tolerance: Double = 0.2,
+    mangleDuplicates: Boolean = false): GenericDataset = {
     importGens(List(file), sampleFile, chromosome, nPartitions, tolerance)
   }
 
   def importGens(files: Seq[String],
     sampleFile: String,
     chromosome: Option[String] = None,
     nPartitions: Option[Int] = None,
-    tolerance: Double = 0.2): GenericDataset = {
+    tolerance: Double = 0.2,
+    mangleDuplicates: Boolean = false): GenericDataset = {
```

Comment:
Need to add the mangle_duplicates argument to GenLoader function call on line 277.